client/gutenberg/editor/controller.js

```diff
@@ -138,6 +138,32 @@ export const redirect = ( { store: { getState } }, next ) => {
 	return page.redirect( `/post/${ getSelectedSiteSlug( state ) }` );
 };
 
+function getPressThisContent( query ) {
+	//also, embed
+	const { text, url, title, image } = query;
+	let content = '';
+
+	//TODO: EMBEDS!
+	// if ( isValidUrl( embed ) ) {
+	// 	pieces.push( ReactDomServer.renderToStaticMarkup( <p>{ embed }</p> ) );
+	// }
+
+	if ( image ) {
+		content = content.concat(
```

Comment:
What's the reason for using `concat` instead of `+=`?

Reply:
no reason, I can change that :D